pom.xml

```diff
@@ -152,6 +152,19 @@
             <plugin>
```

Comment:
Should I also rename the project here?
<name>talend-test-runner</name>
<description>Talend Test Runner</description>
->
<name>jarvis-test-runner</name>
<description>Jarvis Test Runner</description>

Reply:
Done.